src/main/java/uk/gov/register/presentation/dao/Entry.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.register.presentation.dao;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.sql.Timestamp;
```

Comment:
looks like dropwizard-jdbi adds joda time support: https://github.com/dropwizard/dropwizard/blob/master/dropwizard-jdbi/src/test/java/io/dropwizard/jdbi/timestamps/JodaDateTimeSqlTimestampTest.java if this is easy I'd prefer the joda types rather than the java.sql types.